lms/djangoapps/support/views/course_reset.py

```diff
@@ -160,5 +160,5 @@ def post(self, request, username_or_email):
             'display_name': course_enrollment.course_overview.display_name
         }
 
-        reset_student_course.delay(course_id, user.id, request.user.id)
+        reset_student_course.delay(course_id, user.email, request.user.email)
```

Comment:
This is changed because https://github.com/openedx/edx-platform/blob/master/lms/djangoapps/support/tasks.py#L39 requires user emails instead